docs/dh/mocking.md

```diff
@@ -0,0 +1,40 @@
+# Mocking
+
+`app-dh` is using [Mock Service Worker](https://mswjs.io/docs/) also known as MSW to mock our HTTP requests. Mocks are located here:
+`libs/dh/shared/data-access-msw/src/lib/mocks` and registered here: `libs/dh/shared/data-access-msw/src/lib/handlers.ts`.
+
+![Dep graph](./mocking-dep-graph.png)
+
+## Bypass mocks in development
+
+Mocks are enabled by default but can be bypassed by running the `development` target. This will disable the mocks and call the BFF configured in `libs/dh/shared/assets/src/assets/configuration/dh-api-environment.local.json`.
+
+`yarn nx run app-dh:serve:development`
+
+## Add new mocks
+
+Mocks should be added here `libs/dh/shared/data-access-msw/src/lib/mocks` and you can find information about how to mock here.
+After your mocks has been created you need to import your mocks here `libs/dh/shared/data-access-msw/src/lib/handlers.ts`
+
+You can find more information of how to define mocks [here](https://mswjs.io/docs/getting-started/mocks/rest-api).
+
+## Testing
+
+Mocks are shared between the app and testing environment. This means that you usually don't need specific mocks in your tests as these should be testable manually aswell by running the application.
```

Comment:
question: Can you clarify what "as these should be testable manually as well" means? Manually in what way?